misc/build.func

```diff
@@ -491,13 +582,14 @@ install_script() {
   NEXTID=$(pvesh get /cluster/nextid)
   timezone=$(cat /etc/timezone)
   header_info
-  if (whiptail --backtitle "Proxmox VE Helper Scripts" --title "SETTINGS" --yesno "Use Default Settings?" --no-button Advanced 10 58); then
+  if (whiptail --backtitle "ProxmoxVE Community-Scripts" --title "SETTINGS" --yesno "Use Default Settings?" --no-button Advanced 10 58); then
     header_info
-    echo -e "${BL}Using Default Settings${CL}"
-    default_settings
+    echo -e "${DEFAULT}${BOLD}${BL}Using Default Settings${CL}"
+	base_settings
```

Comment:
```suggestion
    base_settings
```
How is this backward compatible with existing scripts when `default_settings` is removed? Have you checked all 200 scripts that they have same settings as are defined in new `base_settings`?

Reply:
I update all CTS when i merge the PR at the same Moment. I create tomorrow an new PR. Then we need to Block new PR's for 1-2 days

Reply:
Locally im done with 79 Scripts. Think tomorrow or tuesday finish

Reply:
To summarize this change we are moving `default_settings` from the scripts to `build.func` and removing the ability to override settings from scripts (except the few variables)? The solution is less flexible than it was before, also less flexible than https://github.com/community-scripts/ProxmoxVE/pull/322

Reply:
I don't understand what you're trying to tell me. The whole thing is ultra flexible, isn't it?

Reply:
Show me 5 scripts that have values other than the 5/6 overwritable values. Everything is still possible with the advanced settings.

Reply:
I'm just trying to get an understanding of the change being made. It seems like these default settings are never customized per script, so it should be fine to centralize it. We should rename it back to `default_settings` though, since that is what it actually is

Reply:
Or i guess that has to be done after removing default settings from all scripts